src/modules/Dropdown/Dropdown.js

```diff
@@ -1047,8 +1047,13 @@ export default class Dropdown extends Component {
     } else if (hasValue) {
       _text = _.get(this.getItemByValue(value), 'text')
     }
-
-    return <div className={classes}>{_text}</div>
+    let finalRenderedText = null
+    if (classes === 'text') {
```

Comment:
I don't add them because I don't know when are used the other cases. If you can explain me them, I can notice if they are needed or not, and also, which ones.